common-test/src/main/java/io/customer/commontest/core/BaseTest.kt

```diff
@@ -51,7 +50,6 @@ abstract class BaseTest {
         get() = di.jsonAdapter
 
     protected lateinit var mockWebServer: MockWebServer
-    protected lateinit var dateUtilStub: DateUtilStub
```

Comment:
Removed so this can safely be moved to `tracking`, as we no longer rely on this for dates.